### PR TITLE
Added Test Case for All Selection Nodes.

### PR DESCRIPTION
**Pull request by riteshchandawar**
This is very long test but if it fails it will not take long time to debug. I have given clear names to each verification point so that we will be easily point out what is the cause of Failure.

Task related to this work [MAGN-4362](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-4362)

<h3>Reviewer</h3>
- [x] @Randy-Ma 

Will issue separate PR for **Revit2015** and **Revit2016**

Test Case is passing on latest build.
image

### CODE REVIEW

File: test/Libraries/RevitIntegrationTests/SelectionTests.cs [hunk 1]
```diff
@@ -362,6 +355,126 @@ public void SelectionInSyncWithDocumentOperations_References()
...
+            var selectFacesNodeID = "00a2189b-88f4-4e15-b9f0-efefa6357b37";
+            AssertPreviewCount(selectFacesNodeID, 5);
+            for (int i = 0; i <= 4; i++)
+            {
+                var surface = GetPreviewValueAtIndex(selectFacesNodeID, i) as Surface;
+                Assert.IsNotNull(surface);
+            }
+
+            // StructuralFramingTypes
+            var structuralFramingTypesNodeID = GetPreviewValue("b786fdf5-b6fa-4aab-a2fd-b40ac12b8b87");
```
Comment (Randy-Ma) [inline test/Libraries/RevitIntegrationTests/SelectionTests.cs]: Is the value of this node an ID? If not, why not change the variable name to be a name easier to understand.

Comment (Randy-Ma) [inline test/Libraries/RevitIntegrationTests/SelectionTests.cs]: Similarly, there are few names which can be changed after this line.

Comment (riteshchandawar) [inline test/Libraries/RevitIntegrationTests/SelectionTests.cs]: @Randy-Ma , those are Node GUID so I have appended node name to understand ID coming fomr which node

**Comment by Randy-Ma** [comment]
@riteshchandawar Thanks for the fix!
It will be easier for others to understand the code if the name of some variables are changed. But other than this, LGTM!

**Pull request merged**